src/game/Creature.cpp

```diff
@@ -1831,7 +1834,7 @@ void Creature::SendAIReaction(AiReaction reactionType)
 void Creature::CallAssistance()
 {
     // FIXME: should player pets call for assistance?
-    if (!m_AlreadyCallAssistance && getVictim() && !isCharmed())
+    if (!m_AlreadyCallAssistance && AllowCallAssistance() && getVictim() && !isCharmed())
```

Comment:
AllowCallAssistance seem not trivial name for me. I would like something more natural like CanCallAssisstance or AllowedToCallAssisstance.